lib/travis/api/v3/billing.rb

```diff
@@ -0,0 +1,56 @@
+module Travis::API::V3
+  class Billing
+    attr_reader :user_id, :subscription_id
+
+    def initialize(user_id = nil, subscription_id = nil)
+      @user_id = user_id
+      @subscription_id   = subscription_id
+    end
+
+    def create_subscription(subscription_params)
+      post('/subscriptions', {user_id: user_id, subscription: subscription_params})
```

Comment:
The `user_id` doesn't need to go in the params (because it should go in the headers, see comment bellow)